src/mono/wasm/debugger/tests/debugger-test/debugger-test.csproj

```diff
@@ -8,6 +8,8 @@
     <WasmGenerateAppBundle>true</WasmGenerateAppBundle>
     <OutputType>library</OutputType>
     <WasmEmitSymbolMap>true</WasmEmitSymbolMap>
+    <!-- some test require full ICU data, force it -->
+    <WasmIncludeFullIcuData>true</WasmIncludeFullIcuData>
```

Comment:
Which tests need this? Can we update those tests to work with sharding?
I think we should be using the defaults, and making sure that those work well.

Reply:
e.g. https://github.com/ilonatommy/runtime/blob/9b76c28567640e4cbe0d20e18b765b8f1a47473f/src/mono/wasm/debugger/DebuggerTestSuite/DateTimeTests.cs#L22
test locales from EFIGS + CJK (if we were to use shards). Loading full ICU allows it to pass.

Reply:
Theoretically I could add locale argument to one of `DebuggerTestBase` constructors and split that CheckDateTimeLocale into two different tests.

Reply:
Oh, sorry, I thought this was also in WBT. lemme look at this again.

Reply:
Good idea. I made `lang=en-US` the default mode with an option of overwriting. Only `DateTimeTests` is problematic, all the rest passes in the default one.
For Chrome `lang` option is respected and works fine, however I could not make the firefox's equivalent `UILocale` work properly, so for now I disabled CJK test on Firefox.